### PR TITLE
Standardize on 'const X' instead of 'X const', part II

### PR DESCRIPTION
**Pull request by tamiko**
In reference to #15908 and #15903

### CODE REVIEW

File: include/deal.II/base/tensor.h [hunk 1]
```diff
@@ -894,7 +894,7 @@ class Tensor
 
   /**
    * This constructor is for internal use. It provides a way
-   * to create constexpr constructors for Tensor<rank, dim, Number>
+   * to create constexprructors for Tensor<rank, dim, Number>
```
Comment (tamiko) [inline include/deal.II/base/tensor.h:897]: :-D

**Review by kronbichler — APPROVED** [review]

**Comment by drwells** [comment]
Can you rebase? Let's make sure everything is squared away this time :)

**Review by masterleinad — APPROVED** [review]

**Review by kronbichler — APPROVED** [review]

**Comment by bangerth** [comment]
Oh, that's a lot of places. Nice work!

**Pull request merged**